shell/platform/embedder/BUILD.gn

```diff
@@ -235,6 +235,23 @@ if (enable_unittests) {
       "//third_party/skia",
     ]
 
+    _label = "//flutter/lib/snapshot:vm_isolate_snapshot.bin"
+    _gen_dir = get_label_info(_label, "target_gen_dir")
+
+    vm_snapshot_data = "${_gen_dir}/vm_isolate_snapshot.bin"
```

Comment:
prefix these 4 with an underscore to indicate that they are private variables.

Reply:
Done.